src/Symfony/Component/Intl/Tests/DateFormatter/AbstractIntlDateFormatterTest.php

```diff
@@ -409,7 +409,11 @@ public function testFormatWithDateTimeZoneGmtOffset()
 
     public function testFormatWithIntlTimeZone()
     {
-        if (PHP_VERSION_ID < 50500 && !(extension_loaded('intl') && method_exists('IntlDateFormatter', 'setTimeZone'))) {
+        if (!extension_loaded('intl')) {
+            $this->markTestSkipped('Int extension not installed.');
+        }
+
+        if (PHP_VERSION_ID < 50500 && !method_exists('IntlDateFormatter', 'setTimeZone')) {
```

Comment:
there are two ways to get this method: either have intl with PHP5.5, or use pecl-intl v3 (could be on 5.3)
so no, it's not the same...